modules/paddlelite/Kconfig

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+config PADDLE_LITE
+	bool "Paddle Lite Support"
+	select REQUIRES_FULL_LIBCPP
```

Comment:
Not sure of our policy on libraries that require C++

Reply:
We wrote it with reference to tflite-micro, because it requires many features of C++.